kubernetes/celery-beat/deployment.yml

```diff
@@ -0,0 +1,51 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: celery-beat-deployment
+  labels:
+    app: celery-beat
+spec:
+  replicas: {{.Values.celeryBeat.replicaCount}}
+  strategy:
+    type: Recreate
+  selector:
+    matchLabels:
+      app: celery-beat
+  template:
+    metadata:
+      labels:
+        app: celery-beat
+    spec:
+      containers:
+      - name: celery-beat
```

Comment:
[Importance 4/10]

Ideally we should know if the pod is up and restart if not
we can use https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/

Reply:
The links we looked at only works for celeryworkers, it doesn't serve as a check for celerybeat. (I tested this out on the test prod server manually killing or suspending celerybeat and seeing what happened).

So I added the liveness probe for the worker, but for celerybeat for now I give up and just let it fail if it fails.